docs/source/enterprise-guide/change-management.mdx

```diff
@@ -0,0 +1,468 @@
+---
+title: Change Management in Apollo Federation
+sidebar_title: Change Management
+---
+
+A federated GraphQL architecture is a multi-tiered architecture made up of
+independent components. GraphQL requests flow through multiple components to
+construct a response, which in turn flows back through those same components, as
+illustrated below:
+
+```mermaid
+graph LR
+    X(Client) --> A(Gateway)
+    A -.-> X
+    A --> B(Subgraph A)
+    B -.-> A
+    A --> C(Subgraph B)
+    C -.-> A
+    B --> D(REST API)
+    D -.-> B
+    D --> E(Database)
+    E -.-> D
+    C --> F(gRPC API)
+    F -.-> C
+    F --> G(Third party API)
+    G -.-> F
+```
+
+The ability to update components independently is a key benefit of federation.
+However, understanding and de-risking changes to a component can be challenging,
+especially as changes may cascade and require changes in other components. This
+is especially true for GraphQL schema defintions, which must be coordinated
+between subgraph services and the gateway.
+
+This section covers the seven most common change management scenarios in a
+federated architecture. We'll consider the proper sequencing of release events,
+how Apollo tooling makes managing changes easier, and when progressive delivery
+techniques like blue-green deploys are useful.
+
+Specifically, we'll examine scenarios that:
+
+**Involve a schema change**
+
+- [Backwards-compatible subgraph schema changes](#backwards-compatible-subgraph-schema-changes)
+- [Backwards-_incompatible_ subgraph schema changes](#backwards-incompatible-subgraph-schema-changes)
+
+**Involve schema changes in multiple subgraphs**
+
+- [Changes to shared types](#changes-to-shared-types)
+- [Changes to query planning](#changes-to-query-planning)
+
+**Don't involve schema changes**
+
+- [Changes to a data source](#changes-to-a-data-source)
+- [Changes to subgraph resolvers](#changes-to-subgraph-resolvers)
+- [Changes to gateway logic/configuration](#changes-to-gateway-logic-or-configuration)
+
+The release guidelines for each scenario should generally apply to all release
+management systems, whether we're deploying to Kubernetes, serverless, or bare
+metal. Some implementation details may change depending on whether we have a
+manual deployment process, a continuous delivery process, or something in
+between.
+
+## Backwards-compatible subgraph schema changes
+
+A schema change is backwards-compatible if it doesn’t affect existing
+operations. Some examples of backwards-compatible changes:
+
+- Adding a type or field — unused until a client includes the fields in new
```

Comment:
Recommend parentheses instead of dashes:

* Adding a type or field (unused until a client includes...)

Reply:
Also recommend removing punctuation from the ends of these list items (punctuate complete sentences, don't punctuate fragments such as these)